views/base.html

```diff
@@ -18,340 +19,340 @@
-                          <div class="pointer-events-none absolute inset-y-0 left-0 flex items-center px-2 text-gray-700">
-                            <svg class="w-6 h-6 fill-primary" aria-label="{{__('Search content')}}"><use xlink:href="#search" /></svg>
-                          </div>
-                          <div class="pointer-events-none absolute inset-y-0 right-0 flex items-center px-2 text-gray-700">
-                            <button class="flex-shrink-0 text-sm border-4 text-primary py-1 px-2 rounded" type="button">
+                                                </div>
+                                            </div>
+
+                                            <div id="searchSelect2Div" class="searchSelect2Div relative">
+                                                <input id="searchSelect2" aria-label="{{__('Search website')}}" readonly type="text" class="searchSelect2 cursor-pointer hover:bg-secondary focus:bg-secondary border-2 border-solid pl-10 pr-48 py-3 lg:text-xl border-secondary w-full pointer-events-auto"
+                                                />
+                                                <div class="pointer-events-none absolute inset-y-0 left-0 flex items-center px-2 text-gray-700">
+                                                    <svg class="w-6 h-6 fill-primary" aria-label="{{__('Search content')}}"><use xlink:href="#search" /></svg>
+                                                </div>
+                                                <div class="pointer-events-none absolute inset-y-0 right-0 flex items-center px-2 text-gray-700">
+                                                    <button class="flex-shrink-0 text-sm border-4 text-primary py-1 px-2 rounded" type="button">
                               {{__('Search website')}}
                             </button>
-                          </div>
-                        </div>
-                      </div>
+                                                </div>
+                                            </div>
+                                        </div>
+                                    </div>
+                                </div>
+
+                            </li>
+                        </ul>
+                    </nav>
+                </div>
+            </header>
+            <!-- End of page header -->
+
+            {% block content %}{{content}}{% endblock %}
+        </div>
+
+        <!-- Footer -->
+        <footer class="site-footer bg-primary text-white-75 mt-10 py-gutter text-sm">
+            <div class="container mx-auto md:flex md:justify-between">
+                <div class="px-gutter pb-6 md:p-gutter md:w-1/3 md:flex flex-col justify-between">
+                    <a class="block w-24 mb-10 h-10 text-white-75 hover:text-white" href="https://ckan.org/">
+                        <span>Powered by</span> {% include "./partials/ckan-logo.svg" %}
+                    </a>
```

Comment:
@virgoaugustine just making sure, these are all due to changes in indentation right?

Reply:
Yes that’s right  @MuhammadIsmailShahzad

Reply:
ok thank.

LGTM!